WIP: Survey Review Checklists (#29)
dime-worldbank/dimeanalytics: survey-review-checklists/data_assurance.tex

		colback = fontcolor,
		sharp corners,
		halign = flush center,
		valign = center,
		height = 0.2\textwidth,
		after skip = 1cm]
		#1
	\end{tcolorbox}
}

%\input{checklist-header.tex}

% ------------------------------ End of preamble ---------------------------------------------
\begin{document}

	\titleBox{
			\textcolor{white}{\Large{DIME Analytics Data Quality Assurance Checklist}}
	}

		\section{Objective}
		In the process of acquiring data, especially primary data, many factors could lead to poor data quality. A plan to ensure quality of data is one that identifies the potential issues in the data, and tracks data quality throughout the process of acquiring data.
	
	\vspace{5mm} %5mm vertical space	
		\section*{Procurement}

\newcounter{row}

\newcommand{\makerow}[1]{%
	% #1 = text field
	#1 &
	\stepcounter{row}%
	\mbox{\CheckBox[print,name=YES\therow, width=0.7em, height=0.7em]{}} &
	\mbox{\CheckBox[print,name=NO\therow, width=0.7em, height=0.7em]{}} &
	\mbox{\CheckBox[print,name=N/A\therow, width=0.7em, height=0.7em]{}} \\
}

\begin{Form}
	\noindent
	\begin{tabularx}{\textwidth}{Xccc}
		\toprule
		\textbf{CHECKS} & YES & NO & N/A \\
		\midrule

> 		\makerow{Does the TOR establish \textbf{minimum quality indicators} required for the data to be considered acceptable?}

Reviewer: I'm actually not sure offhand what minimum quality indicators would be.

		\midrule

> 		\makerow{Does the TOR establish a \textbf{maximum response time} for the survey firm to address data quality issues identified?}

Reviewer: perhaps add, does the TOR establish a clear protocol for identifying and correcting data quality issues? maximum response time is part of that for sure, but establishing that the team will be running quality checks and the survey firm is expected to respond seems the most critical.

		\bottomrule
	\end{tabularx}
	
	\vspace{5mm} %5mm vertical space		

		\section*{Survey Design}

\begin{tabularx}{\textwidth}{Xccc}
\toprule
\textbf{CHECKS} & YES & NO & N/A \\
\midrule
\makerow{Does the survey have \textbf{relevance and constraint logic} set up to aid the enumerator?}
\midrule
\makerow{Does the survey have an \textbf{an ID variable} to identify respondents and link them to the respondents database?}
\midrule

> \makerow{Does the survey have an \textbf{unique ID generator} for each survey submission  (if survey is being done over SurveyCTO this is automatically generated as the ‘key’ variable.}

Reviewer: this seems second-order to me (since ID var is already established)

Author: This is second-order, yes but comes in handy when dealing with duplicate submissions with the same ID variable as it allows for easy management of duplicates.

\midrule

> \makerow{Does the survey have \textbf{duration spent} on completing a survey and modules within the survey?}

Reviewer: ```suggestion
\makerow{Does the survey include calculation of \textbf{duration} of the full survey and each module within the survey?}
```
```suggestion
\makerow{Does the survey have \textbf{duration spent} on completing a survey and modules within the survey?}
```

Reviewer: the calculation by module also seems second-order. useful for long surveys for sure. but not always critical.

\midrule

> \makerow{Has the \textbf{translation} been approved?}

Reviewer: by whom? maybe instead, was the survey fully translated into all languages used for data collection?

\midrule

> \makerow{Does \textbf{enumerator training materials} include explanation of informed consent, Q\&A session, mock interviews and review of best practices?}

Reviewer: suggest including something about the enumerator manual here as well, like does one exist and does it explain all questions and survey conventions. 
maybe consolidate q&a/mock interviews to does the agenda for the enumerator training include both classroom and field practice, and allocate sufficient time for understanding the survey instrument?

\midrule
\makerow{In case of projects with multiple survey instruments, does each survey have it’s own detailed survey manual?}

\bottomrule
\end{tabularx}

\vspace{5mm} %5mm vertical space

		\section*{High-Frequency Checks (HFC)}

\noindent
\begin{tabularx}{\textwidth}{Xcccc}
\toprule
\textbf{CHECKS} & YES & NO & N/A \\
\midrule

> \makerow{Is there a plan in place to conduct \textbf{high-frequency checks} on the data being collected?}

Reviewer: this question seems like the most important, the rest are good to document but not as critical. perhaps in general we want to think about setting this up as minimum expectation and then recommended best practices? not sure about the best way to do that, just thinking out loud ...

\midrule

> \makerow{Does the HFC \textbf{monitor survey duration} and start/end times and day of the week of a survey fill-out?}

Reviewer: how does day of the week affect quality? maybe this is getting at more generally whether the HFCs monitor compliance with established field protocols re: interview duration and timing? might want to add here whether HFCs check for enumerator-specific patterns (e.g. an enumerator that is super slow or consistently doing interviews outside of expected working hours)

\midrule

> \makerow{Do the HFCs check for key treatment and \textbf{outcome variables?}}

Reviewer: check what for them? distribution?

\midrule

> \makerow{Do the HFCs check for \textbf{sample completeness?}}

Reviewer: meaning response rate / attrition?

\midrule
\makerow{Do the HFCs report \textbf{duplicates, missing values and errors} in logic that are not automatically corrected by the survey form?}
\midrule
\makerow{Is the process for generating flags from HFCs \textbf{easy and fast to run} on a daily basis for FCs?}
\midrule

> \makerow{Does the HFC process account for potential lack of \textbf{access to WiFi} or electricity for team on the field?}

Reviewer: this is very specific. perhaps more generalizable, is there a protocol for sharing HFC results with teams that accounts for local conditions (e.g. limited connectivity) and accounts for safe sharing of personal data?

\midrule

> \makerow{Do HFCs monitor geographic location of projects based on \textbf{GPS coordinates?}}

Reviewer: this wording is quite vague. is it getting at whether HFCs check whether the interview was actually conducted in/near the expected location?

\midrule

> \makerow{Has the HFC plan been communicated to all \textbf{stakeholders} in the team (incl. survey firm) with enough time before survey round?}

Reviewer: this could be incorporated with the question about protocol i suggested in my comment above.

\bottomrule
\end{tabularx}

\vspace{5mm} %5mm vertical space

\section*{Backchecks and audits}

\noindent
\begin{tabularx}{\textwidth}{Xcccc}
\toprule
\textbf{CHECKS} & YES & NO & N/A \\
\midrule

> \makerow{Is there a plan in place to conduct \textbf{back-checks} on the data collected?}

Reviewer: perhaps have a space here for people to indicate what proportion of the sample was back-checked?

Reviewer: we do a combo of checklist plus spaces for open-ended responses in the reproducibility package checklist and that seems to work pretty well

\midrule

> \makerow{Does the back-checks plan include \textbf{variables unlikely to change}, as well as variables that stem from difficult or lengthy modules? }

Reviewer: ```suggestion
\makerow{Does the back-checks survey instrument include \textbf{variables unlikely to change}, as well as variables that stem from difficult or lengthy modules? }
```
```suggestion
\makerow{Does the back-checks plan include \textbf{variables unlikely to change}, as well as variables that stem from difficult or lengthy modules? }
```

\midrule

> \makerow{Is there a plan to audit through \textbf{audio recordings}? If so, is it mentioned in the informed consent?}

Reviewer: this question seems out of order because it comes in the middle of questions about backchecks

\midrule
\makerow{Does the back-checks plan include back-checking \textbf{all enumerators?}}
\midrule
\makerow{Does the back-check plan include back-checking \textbf{20 percent of the sample} in the first 2 weeks and 10 percent of the sample in the following weeks?}
\bottomrule
\end{tabularx}

		\section*{Communication and Field Management}

\noindent
\begin{tabularx}{\textwidth}{Xcccc}
	\toprule
	\textbf{CHECKS} & YES & NO & N/A \\
	\midrule

> 	\makerow{Does team have a plan for \textbf{communication and data quality checks feedback} with enumerators?}

Reviewer: this seems potentially redundant with the questions earlier about protocols for HFCs

	\midrule

> 	\makerow{Is there a protocol in place for the field team to \textbf{gather information and respond to flags} raised during data quality checks?}

Reviewer: potentially redundant with HFCs question, unless this is also meant to include backchecks/audits?

	\midrule

> 	\makerow{Has the team put together \textbf{tracking systems} for enumerators to fill out as part of their daily tasks?}

Reviewer: meaning like a supervisor logbook of interviews completed by the team?

	\midrule
	\makerow{Does the team have a plan for \textbf{reconciling data} from the tracking system with the responses on the server}
	\midrule

> 	\makerow{If feasible in the context, does the team plan on research staff \textbf{accompanying enumerator}? }

Reviewer: this seems like it would fit better in the sections on backchecks / audits
